x-pack/plugins/canvas/types/elements.ts

```diff
@@ -18,6 +18,15 @@ export interface ElementSpec {
   filter?: string;
   width?: number;
   height?: number;
+  /**
+   * the flag shows if the expression is defined at the external plugin
+   */
+  external?: boolean;
```

Comment:
Let's remove this, and the disabled plugin check.  For now, we are ok with the element "breaking" if the plugin is disabled.

Reply:
Ok, thanks.

Reply:
Done.